TrackingTools/TrajectoryFiltering/interface/CkfBaseTrajectoryFilter.h

```diff
@@ -51,6 +51,18 @@ class CkfBaseTrajectoryFilter : public TrajectoryFilter {
   virtual bool toBeContinued( TempTrajectory& traj) const {return TBC<TempTrajectory>(traj);}
 
   virtual  std::string name() const { return "CkfBaseTrajectoryFilter";}
+
+  inline edm::ParameterSetDescription getFilledConfigurationDescription() {
+    edm::ParameterSetDescription descLooper           = theLooperTrajectoryFilter->getFilledConfigurationDescription();
+    edm::ParameterSetDescription descLostHitsFraction = theLostHitsFractionTrajectoryFilter->getFilledConfigurationDescription();
+    edm::ParameterSetDescription descMinHits          = theMinHitsTrajectoryFilter->getFilledConfigurationDescription();
+
+    edm::ParameterSetDescription desc;
+    desc.add<edm::ParameterSetDescription>("looperTrajectoryFilter",          descLooper);
+    desc.add<edm::ParameterSetDescription>("lostHitsFractionTrajectoryFilter",descLostHitsFraction);
+    desc.add<edm::ParameterSetDescription>("minHitsTrajectoryFilter",         descMinHits);
+    return desc;
+  }
```

Comment:
Out of curiosity, what is this function for? Are we going to get a `fillDescriptions()` mechanism for global PSets?

If I have understood things correctly, I'm also afraid that the code above would generate a PSet that would not be valid for CkfBaseTrajectoryFilter, that is
```
cms.PSet(
    looperTrajectoryFilter = cms.PSet(
        ...
    ),
    lostHitsFractionTrajectoryFilter = cms.PSet(
        ...
    ),
    minHitsTrajectoryFilter = cms.PSet(
        ...
    )
)
```
(although personally I wouldn't mind this style)